spec/schema.json

```diff
@@ -267,6 +267,9 @@
           "type": "string",
           "format": "uuid"
         },
+        "http_request_id": {
```

Comment:
Is this needed / ever going to be used by the client? Understand that it will be stored in Mongo DB but do we need to expose this internal information through the public interface?

Reply:
i guess not. i'll close.